modules/misc/test/tests/kernels/diffusion/tests

```diff
@@ -0,0 +1,39 @@
+[Tests]
+  [./1D_transient]
+    type = 'Exodiff'
+    input = '1D_transient.i'
+    exodiff = '1D_transient_out.e'
+    abs_zero = 1e-4
```

Comment:
Sorry for not commenting on this in my initial review, but could we up this to 1e-7 or 1e-8? 1e-4 is starting to get a little high for my liking (unless there was an explicit reason you set this to this value)

Reply:
Yeah, thats what I was wondering. It was necessary to pass on my mac.
```
error  rel diff:  4.1893557e-05 ~  4.1894108e-05 = 1.31568e-05 (FAILED)
```

I almost think that error shouldnt be included in the exodiff

Reply:
Also, I'm kind of surprised the error is that high. I already reduced the time steps and time progression, and got it down from 1e-3

Reply:
@lindsayad It matches the analytical expression...it just seems like high error. I removed exodiff, but is the high error okay?

Reply:
I think `error` should be included because it is kind of the central element of the test; the test is meant to verify that the numerical solution matches the analytical solution. As to whether `4e-5` is high...I'm going to defer to the writer of the test on that one :smile: I guess in my experience for 10s of elements in a given direction  order `1e-5` doesn't seem out of the ordinary so I would be inclined to accept it. 

As to handling the exodiff itself...I think the right option here is `rel_err` which defaults to `5.5e-6`. I would decrease `abs_zero` to 1e-8 and then set `rel_err` equal to `2e-5`

Reply:
This test was taken from the heat conduction analytical expressions from wayyy back when. It seems like changes in the error are not as important as changes in the variables, which is captured by the exodiff. The error magnifies those changes by comparing it to a similar number, resulting in big swings if the variables are incorrect. I don't want to overshadow changes in the variables with loose tolerances just to make the error match right? This could be a custom cmp problem....or just removing the error from the exodus file altogether like I did here?

Reply:
@lindsayad It does beg the question how are you capturing the error is low enough. It's easy enough to see that the error is being calculated, it's a bit harder to make sure the error is low beyond trusting the person who created the test.